ArduPlane/quadplane.cpp

```diff
@@ -848,7 +848,8 @@ void QuadPlane::multicopter_attitude_rate_update(float yaw_rate_cds)
     check_attitude_relax();
 
     // normal control modes for VTOL and FW flight
-    if (in_vtol_mode()) {
+    // tailsitter in transition to VTOL flight is not really in a VTOL mode yet
+    if (in_vtol_mode() && !in_tailsitter_vtol_transition()) {
```

Comment:
what effect does this have if qassist is not enabled?

Reply:
None, it sets the desired rates but the attitude controller is never run.

Reply:
Wow, that *is* confusing..
It looks to me like this delays the transition from FW to MC controller until the transition is complete, regardless of whether qassist is active.

Reply:
It changes the what we ask the MC controller to do regardless of Qassist. But that only makes a difference if you call motors output somewhere, and that is only done in Qassist.

Yeah its very confusing but I cant see a nice path to tidying it up, Tailsitters have special cases dotted all over.

Reply:
I inserted some log records to trace calls to motors_output, and the MC rate updates and rate controller, and everything looks good to me with qassist disabled:
![image](https://user-images.githubusercontent.com/2300221/95602666-78f5dd00-0a12-11eb-84c3-60ab03d09dcb.png)

Reply:
Is there currently any way to get the quadplane transition_state from the log?

Reply:
Yes, I just added it to QTUN, https://github.com/ArduPilot/ardupilot/pull/15432